keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -678,14 +678,15 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
   Whether it is safe to use the RequestWatchProgress API call 
   or if the experimental-watch-progress-notify-interval flag has been set.
   Knowing etcd configuration will be used to automatically disable the streaming feature.
+- Use WatchProgressRequester to request progress notifications directly from etcd.
+  This mechanism was developed in [Consistent Reads from Cache KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#use-requestprogress-to-enable-automatic-watch-updates)
+  and will reduce the overall latency for watchlist requests.
 
 #### GA
-- Consider using WatchProgressRequester to request progress notifications directly from etcd.
-  This mechanism was developed in [Consistent Reads from Cache KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache#use-requestprogress-to-enable-automatic-watch-updates)
-  and could reduce the overall latency for watchlist requests.
 - [Switch](https://github.com/kubernetes/kubernetes/blob/a07b1aaa5b39b351ec8586de800baa5715304a3f/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L416) 
   the `storage/cacher` to use streaming directly from etcd 
   (This will also allow us to [remove](https://github.com/kubernetes/kubernetes/blob/a07b1aaa5b39b351ec8586de800baa5715304a3f/staging/src/k8s.io/client-go/tools/cache/reflector.go#L110) the `reflector.UseWatchList` field).
+- Make  **list** calls expensive in APF.
```

Comment:
This should be post GA.  clients need time to update.

Reply:
Do we want to capture it as a new section `Post-GA` (will not match the template) or will it be okay to update the text ?

Reply:
Let's add "Post-GA' section for this

Reply:
> Let's add "Post-GA' section for this

Yes.  Spoke with Lucasz this morning and explained that we need to ensure the fallback list behavior works until all supported releases have streaming list GA and we have turned on "lock to on".

Reply:
yes, thanks, pushed.